amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/polling/PollingConfig.java

```diff
@@ -86,11 +88,15 @@ public PollingConfig(String streamName, KinesisAsyncClient kinesisClient) {
      * The value for how long the ShardConsumer should sleep in between calls to
      * {@link KinesisAsyncClient#getRecords(GetRecordsRequest)}.
      *
+     * If this is not set using {@link PollingConfig#idleTimeBetweenReadsInMillis},
+     * it defaults to 1500 ms.
+     *
      * <p>
-     * Default value: 1000L
+     * Default value: 1500L
      * </p>
      */
-    private long idleTimeBetweenReadsInMillis = 1000L;
+    @Setter(AccessLevel.NONE)
+    private long idleTimeBetweenReadsInMillis = 1500L;
```

Comment:
Why is it we are chaniging this default?

Reply:
Because the current default of 1000L is never used, if it is not explicitly configured it used a default defined in an internal API which is 1500L, so I am keeping them consistent without behavior change.